src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/KnownOwnerViewModel.cs

```diff
@@ -8,17 +8,14 @@ namespace NuGet.PackageManagement.UI.ViewModels
 {
     public class KnownOwnerViewModel
     {
-        private string _name;
-        private Uri _link;
-
         public KnownOwnerViewModel(KnownOwner knownOwner)
         {
-            _name = knownOwner.Name;
-            _link = knownOwner.Link;
+            Name = knownOwner.Name;
+            Link = knownOwner.Link;
```

Comment:
Since this class is public, we should guard against knownOwner being null since we dereference it without knowing if a null was passed in the constructor.

```suggestion
            if (knownOwner == null)
            {
                throw new ArgumentNullException(nameof(knownOwner);
            }
            
            Name = knownOwner.Name;
            Link = knownOwner.Link;
```

Reply:
@donnie-msft did you see this feedback?

Reply:
@jebriede I did and will address this one. The others about style I probably will not, because I prefer the readability of it. I'm pretty sure the IL is the same in those cases.

Reply:
Think this should do it! https://github.com/NuGet/NuGet.Client/pull/5784